locales/de-de.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Was du bist? (Wähle jetzt aus, du kannst es später ändern)",
+	"select-user-type.driver": "Fahrer",
+	"select-user-type.passenger": "Passagier",
+	"request-phone.type_your_phone": "Bitte sende uns deine Telefonnummer. Wir geben diese Nummer nicht ohne Erlaubnis weiter.",
+	"request-phone.button_text": "Nummer senden",
+	"request-phone.all_set": "👌 Alles klar soweit, du kannst jetzt eine Fahrt bestellen!",
+	"passenger-index.choose_taxi_type": "Welche Art von Taxi willst du?",
+	"passenger-index.car": "🚗 Auto",
+	"passenger-index.motorbike": "🏍 Roller / Motorrad",
+	"passenger-index.s": "🔧 Einstellungen",
+	"passenger-index.error_only_known_type": "Unbekannte Auswahl, bitte wähle das Zweirad, Auto oder die Einstellungen aus.",
+	"passenger-request-location.provide_location": "Bitte teile deinen Standort, um Fahrer im Umkreis zu finden.",
+	"passenger-request-destination.provide_destination": "Wohin möchtest du? Bitte gebe die genaue Adresse an (damit der GPS-Navigator es versteht) oder beschreibe den Ort.",
+	"order-submitted.order_submitted": "⏳ Suche nach Fahrern... Wenn du eine Fahrt gefunden hast, klicke auf die Taste unten 👇",
+	"order-submitted.cancel": "Suche beenden",
+	"order-submitted.error_incorrect_input": "Unbekannte Auswahl, bitte versuche es erneut.",
+	"order-cancelled.order_cancelled": "👌 Danke, die Suche nach Fahrern wurde beendet.",
+	"driver-select-vehicle-type.select": "Wähle den Fahrzeugtyp, den du als Taxi verwenden möchtest.",
+	"driver-select-vehicle-type.motorbike": "🏍 Roller / Motorrad",
+	"driver-select-vehicle-type.car": "🚗 Auto",
+	"driver-select-vehicle-type.error_only_known_type": "Unbekannte Auswahl, bitte wähle Auto oder Roller / Motorrad aus.",
+	"driver-explain-checkins.text": "Fast fertig, noch ein paar Dinge zur Beachtung. Telegramm hat keine Echtzeit-GPS-Standortunterstützung. Also jedes Mal, wenn du deinen Standort ändert, checke ein. Oder checke jede Stunde ein, wenn du bereit bist neue Aufträge zu akzeptieren. Das wird uns helfen, Passagiere im Umkreis zu finden. Tippe \"Weiter\".",
```

Comment:
How about? 

Fast fertig, bevor es losgeht gibt es noch ein paar Dinge zu beachten.

Also

>wenn du deinen Standort ändert

-> wenn du deinen Standort änderst

This could also be

Da Telegramm kein Echtzeit-GPS unterstützt, musst du daran denken neu einzuchecken sobald du deinen Standort änderst. Alternativ kannst du auch jede Stunde einchecken wenn du bereit bist neue Aufträge zu akzeptieren.